### PR TITLE
Fix Panzer's worket_builder test with UVM off (2/2)

### PR DESCRIPTION
**Pull request by JacobDomagala**

### CODE REVIEW

**Review by bathmatt — APPROVED** [review]
just letting the auto tester run, don't merge

File: packages/phalanx/src/Phalanx_MDField.hpp
Comment (bathmatt) [inline packages/phalanx/src/Phalanx_MDField.hpp:677]: @rppawlo thoughts on augmenting the kokkos namespace and not just using as_view() at the point of usage?

Comment (JacobDomagala) [reply packages/phalanx/src/Phalanx_MDField.hpp]: Ok, I reverted the changes. Let's be consistent with using `as_view`

**Comment by JacobDomagala** [comment]
Fixed the failing tests, this should be ready to merge if there're no issues.

**Review by bathmatt — APPROVED** [review]

**Pull request merged**
